Firmware/MotorControl/motor.cpp

```diff
@@ -66,6 +70,9 @@ bool Motor::apply_config() {
     config_.parent = this;
     is_calibrated_ = config_.pre_calibrated;
     update_current_controller_gains();
+    fet_thermistor_.motor_ = this;
+    motor_thermistor_.motor_ = this;
```

Comment:
This would be more suitable to put in the Motor constructor (it doesn't need to re-run when reloading the config).

Reply:
Addressed in fd848f2